docs/interactive_engine/tutorial_ldbc_gremlin.md

```diff
@@ -831,3 +831,1087 @@ At last, we would like to introduce `endV()` step in path expansion. Since a pat
 
 Figure 18. The examples of path expansion.
 :::
+### Real Applications
+
+Graph traversal enables us to conduct more complicated data analytics on the graph. 
+
+For example, the path `comment-hasCreator->person->isLocatedIn->city-isPartOf->country` can help to determine the nationality of `comments`. Therefore,  we can calculate the histogram of comment length for comments from China easily with GIE:
+
+```python
+import matplotlib.pyplot as plt
+import numpy as np
+import graphscope as gs
+from graphscope.dataset.ldbc import load_ldbc
+
+# load the modern graph as example.
+graph = load_ldbc()
+
+# Hereafter, you can use the `graph` object to create an `gremlin` query session
+g = gs.gremlin(graph)
+# Extract all comments' contents in China
+q1 = g.execute('g.V().hasLabel(\'person\').where(out(\'isLocatedIn\').out(\'isPartOf\').values(\'name\').is(eq(\'China\'))).in(\'hasCreator\').values(\'content\')')
+comment_contents = q1.all()
+comment_length = [len(comment_content.split()) for comment_content in comment_contents];
+
+plt.hist(x=list(filter(lambda x:(x< 50), comment_length)), bins='auto', color='#607c8e',alpha=0.75)
+plt.grid(axis='y', alpha=0.75)
+plt.xlabel('Comments Length')
+plt.ylabel('Comments Count')
+plt.title('Comment Length Histogram for comments from China')
+plt.show()
+```
+:::{figure-md}
```

Comment:
You can check `https://alibaba-graphscope-build-pr-2583.surge.sh/interactive_engine/tutorial_ldbc_gremlin.html` for the review. These images are not shown. You can include an image as follows:
```
:::{figure-md}
<img src="../images/xx.png"
     alt="xxx"
     width="xx% to fit the preview">

Figure xx. The caption.
:::
```